packages/components/inputs/localized-rich-text-input/src/localized-rich-text-input.visualspec.js

```diff
@@ -1,5 +1,20 @@
 import percySnapshot from '@percy/puppeteer';
+import puppeteer from 'puppeteer';
 import { getDocument, queries } from 'pptr-testing-library';
+let browser;
+let page;
+
+jest.setTimeout(20000);
+
+beforeEach(async () => {
+  browser = await puppeteer.launch({
+    slowMo: 10,
+  });
+  page = await browser.newPage();
+});
+afterEach(async () => {
+  await browser.close();
+});
```

Comment:
This is hardly a solution, but running each of the tests in slow motion makes them pass 🤯 And e.g. with value `5` they don't pass. Seems to be some kind of a race condition but any help with that would be welcome